curriculum/challenges/english/25-front-end-development/review-css-accessibility/671a955b74ab5588735800d1.md

```diff
@@ -34,9 +50,32 @@ Review the concepts below to prepare for the upcoming quiz.
 }
 ```
 
+- **`scroll-behavior: smooth;`**: This property and value enables a smooth scrolling behavior.
+- **`prefers-reduced-motion` feature**: This is a media feature that can be used to detect the user's animation preference.
+
+```css
+@media (prefers-reduced-motion: no-preference) {
+  * {
+    scroll-behavior: smooth;
+  }
+}
+```
+
+In the above example, smooth scrolling is enabled if the user doesn't have animation preference set on their device.
+
+## Hiding Content with HTML Attributes
+
 - **`aria-hidden` attribute**: Used to hide an element from people using assistive technology such as screen readers. For example, this can be used to hide decorative images that do not provide any meaningful content.
 - **`hidden` Attribute**: This attribute is supported by most modern browsers and hides content both visually and from the accessibility tree. It can be easily toggled with JavaScript.
```

Comment:
```suggestion
- **`hidden` attribute**: This attribute is supported by most modern browsers and hides content both visually and from the accessibility tree. It can be easily toggled with JavaScript.
```

in other places it seems we use "attribute" lowercase